examples/README.md

```diff
@@ -115,166 +115,84 @@ kubectl create -f pvc-restore.yaml
 kubectl create -f pod-restore.yaml
 ```
 
-## How to enable multi node attach support for RBD
+## How to test RBD MULTI_NODE_MULTI_WRITER BLOCK feature
+
+Requires feature-gates: `BlockVolume=true` `CSIBlockVolume=true`
+
+*NOTE* The MULTI_NODE_MULTI_WRITER capability is only available for
+Volumes that are of access_type `block`
 
 *WARNING*  This feature is strictly for workloads that know how to deal
 with concurrent acces to the Volume (eg Active/Passive applications).
 Using RWX modes on non clustered file systems with applications trying
 to simultaneously access the Volume will likely result in data corruption!
 
-### Example process to test the multiNodeWritable feature
-
-Modify your current storage class, or create a new storage class specifically
-for multi node writers by adding the `multiNodeWritable: "enabled"` entry to
-your parameters.  Here's an example:
+Following are examples for issuing a request for a `Block`
+`ReadWriteMany` Claim, and using the resultant Claim for a POD
 
-```yaml
-apiVersion: storage.k8s.io/v1
-kind: StorageClass
-metadata:
-   name: csi-rbd
-provisioner: rbd.csi.ceph.com
-parameters:
-    monitors: rook-ceph-mon-b.rook-ceph.svc.cluster.local:6789
-    pool: rbd
-    imageFormat: "2"
-    imageFeatures: layering
-    csiProvisionerSecretName: csi-rbd-secret
-    csiProvisionerSecretNamespace: default
-    csiNodePublishSecretName: csi-rbd-secret
-    csiNodePublishSecretNamespace: default
-    adminid: admin
-    userid: admin
-    fsType: xfs
-    multiNodeWritable: "enabled"
-reclaimPolicy: Delete
 ```
-
-Now, you can request Claims from the configured storage class that include
-the `ReadWriteMany` access mode:
-
-```yaml
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: pvc-1
+  name: block-pvc
 spec:
   accessModes:
   - ReadWriteMany
+  volumeMode: Block
   resources:
     requests:
       storage: 1Gi
-  storageClassName: csi-rbd
 ```
 
 Create a POD that uses this PVC:
 
-```yaml
+```
```

Comment:
I think adding yaml results in a good rendering in markdown files

Reply:
Good point adding the tag to all the yaml items I added, thanks!